org.eclipse.m2e.core.tests/resources/projects/mavenConfig/bundle/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,7 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Bundle
+Bundle-SymbolicName: my.bundle3
```

Comment:
When I change the BSN to `my.bundle` the tests fail when being executed with the other tests of the class, but when executed as only case it passes.

The reason is that when calling `projectRegistry.getProjectFacade(pom);` in `ProjectRegistryManager.createExecutionContext(IProjectRegistry state, IFile pom, IProjectConfiguration resolverConfiguration)` an old facade from another test-case is found that has a different multi-module root, which at that point does not exist anymore.
Looks like the clean-up of the registry is not done (at all or just too late?).